<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 36, 0);
border-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="pronunciationUK">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>WindowsGraphics</normalon>
     <activeon>:/voiceButtonIcon.png/voiceButtonIcon.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_the_main_page_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>300</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>Back to the main page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_word_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="searchbutton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>141</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchfield">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>371</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
font-size:14pt; color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(46, 194, 126);
background-color: rgb(3, 54, 22)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="definitionsText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>540</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
font-size:14pt; color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(46, 194, 126);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pronunciationUS">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>WindowsGraphics</normalon>
     <activeon>:/voiceButtonIcon.png/voiceButtonIcon.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="usageText">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>540</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
font-size:14pt; color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(46, 194, 126);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_the_exerciser_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>110</y>
     <width>300</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>Go to the exerciser</string>
   </property>
  </widget>
  <widget class="QLabel" name="addLine">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="pronunciationUKtext">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>UK</string>
   </property>
  </widget>
  <widget class="QLabel" name="pronunciationUStext">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>US</string>
   </property>
  </widget>
  <widget class="QLabel" name="definitionsTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>Definitions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="usageTitle">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>Usage:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
